src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -35,7 +35,12 @@ namespace Microsoft.CodeAnalysis
     internal partial class SolutionState
     {
         // the version of the workspace this solution is from
-        private readonly int _workspaceVersion;
+        public readonly int WorkspaceVersion;
+
+        public readonly string? WorkspaceKind;
```

Comment:
I admit I preffered this being kept as readonly props, since it's a bit more clear we actually intended to expose it internally.

Reply:
Isn't that conveyed by being them `public`? I can switch all these, including AnalyzerReferences to props if you prefer.